monitoring/main.tf

```diff
@@ -0,0 +1,75 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = ">= 0.11"
```

Comment:
Why `0.11`? We're on v2.0.2 now.

You'd need to add the `coder_monitoring` datasource to the terraform provider before we can merge this, do you have a PR in the works for this?

Reply:
Also is there an RFC design on how `coder_monitoring` will work.?